go/vt/throttler/throttlerz.go

```diff
@@ -73,11 +75,18 @@ func throttlerzHandler(w http.ResponseWriter, r *http.Request, m *managerImpl) {
 
 func listThrottlers(w http.ResponseWriter, m *managerImpl) {
 	throttlers := m.Throttlers()
-	listTemplate.Execute(w, map[string]interface{}{
+	err := listTemplate.Execute(w, map[string]interface{}{
 		"Throttlers": throttlers,
 	})
+	// Log error
+	if err != nil {
+		log.Errorf("listTemplate.Execute(w, map[string]interface{}{\n\t\t\"Throttlers\": throttlers,\n\t}) failed :%v", err)
+	}
 }
 
 func showThrottlerDetails(w http.ResponseWriter, name string) {
-	detailsTemplate.Execute(w, name)
+	// Log error
+	if err := detailsTemplate.Execute(w, name) ; err != nil {
+		log.Errorf("detailsTemplate.Execute(w, name) failed :%v", err)
```

Comment:
rewrite error message.

Reply:
log.Errorf("detailsTemplate failed :%v", err) 
Is this error message alright